scripts/lua/openstreetmap-carto.lua

```diff
@@ -0,0 +1 @@
+../../openstreetmap-carto.lua
```

Comment:
I'd like to get rid of the need for the symbolic link, but I'm not sure how to do that best within Lua.